open_instruct/olmo_adapter/__init__.py

```diff
@@ -0,0 +1,226 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+from torch import nn
+from torch.nn import MSELoss, CrossEntropyLoss, BCEWithLogitsLoss
+from transformers.modeling_outputs import SequenceClassifierOutputWithPast
+from transformers.models.olmo2.modeling_olmo2 import Olmo2Config, Olmo2Model, Olmo2PreTrainedModel
+from transformers.models.olmoe.modeling_olmoe import OlmoeConfig, OlmoeModel, OlmoePreTrainedModel
+
+
+class Olmo2ForSequenceClassification(Olmo2PreTrainedModel):
```

Comment:
Can we link where we got this? I'm guessing its from the OLMo repo?

Reply:
No this is from the transformer subclass. I just implemented it based on how other models are done in transformers.  

In theory this would be better to live in the transformers upstream, but it's a bit of effort going through the community contributions stuff, so I am just doing it in our repo so life is easier.